sfxclient/httpsink.go

```diff
@@ -354,6 +366,27 @@ func mapToProperties(properties map[string]interface{}) []*com_signalfx_metrics_
 	return response
 }
 
+type spanResponse struct {
+	Valid   uint64 `json:"valid"`
+	Invalid struct {
+		InvalidSpanID []string `json:"invalidSpanID"`
```

Comment:
there are a lot more responses than just invalidSpanID. I wouldn't try to create a struct that has them all. Just a map of string:int should be fine and construct that into a reasonable response. 

you can checkout stream.go in sfxinternalgo for all of them if you're super interested.